drools-wb-screens/drools-wb-appformer-extensions-editor/drools-wb-appformer-extensions-editor-client/src/main/java/org/drools/workbench/screens/appformer/extensions/client/AppFormerExtensionsEditorEntryPoint.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.drools.workbench.screens.appformer.extensions.client;
+
+import org.jboss.errai.ioc.client.api.AfterInitialization;
+import org.jboss.errai.ioc.client.api.EntryPoint;
+
+@EntryPoint
+public class AppFormerExtensionsEditorEntryPoint {
```

Comment:
Any need for this class? 

You're not declaring an ```errai-ui``` ```@Bundle``` or anything (although perhaps the client is not i18n'ed... let me look)...